derive_client/constants.py

```diff
@@ -101,6 +103,8 @@ class EnvConfig(BaseModel, frozen=True):
             L1_CHUG_SPLASH_PROXY="0x61e44dc0dae6888b5a301887732217d5725b0bff",
             WITHDRAW_WRAPPER_V2="0xea8E683D8C46ff05B871822a00461995F93df800",
             DEPOSIT_WRAPPER="0x9628bba16db41ea7fe1fd84f9ce53bc27c63f59b",
+            ARBITRUM_DEPOSIT_WRAPPER="0x076BB6117750e80AD570D98891B68da86C203A88",  # unknown address
```

Comment:
This were retrieved from trying to withdrawal from the frontend.

Looks like a mismatched between old and new types for these 2 chains.